internal/runners/packages/packages.go

```diff
@@ -80,15 +95,40 @@ func executePackageOperation(prime primeable, packageName, packageVersion string
 			return errs.Wrap(err, "Could not resolve pkg and namespace")
 		}
 		validatePkg = false
+
+		pg.Stop(locale.T("progress_found"))
 	}
 
 	if strings.ToLower(packageVersion) == latestVersion {
 		packageVersion = ""
 	}
 
+	if validatePkg {
+		pg = output.NewDotProgress(out, locale.Tl("progress_search", "", packageName), 10*time.Second)
+
+		packages, err := model.SearchIngredientsStrict(ns, packageName, false, false)
+		if err != nil {
+			return locale.WrapError(err, "package_err_cannot_obtain_search_results")
+		}
+		if len(packages) == 0 {
+			suggestions, err := getSuggestions(ns, packageName)
+			if err != nil {
+				logging.Error("Failed to retrieve suggestions: %v", err)
+			}
+			if len(suggestions) == 0 {
+				return locale.WrapInputError(err, "package_ingredient_alternatives_nosuggest", "", packageName)
+			}
+			return locale.WrapInputError(err, "package_ingredient_alternatives", "", packageName, strings.Join(suggestions, "\n"))
+		}
```

Comment:
Isn't this https://activestatef.atlassian.net/browse/DX-500?

Reply:
It is. I didn't introduce this code in this PR, I just moved it.